hermes-mock/src/test/groovy/pl/allegro/tech/hermes/mock/HermesMockAvroTest.groovy

```diff
@@ -0,0 +1,76 @@
+package pl.allegro.tech.hermes.mock
+
+import org.apache.avro.Schema
+import org.apache.avro.reflect.ReflectData
+import org.junit.Rule
+import pl.allegro.tech.hermes.test.helper.endpoint.HermesPublisher
+import spock.lang.Specification
+
+class HermesMockAvroTest extends Specification {
```

Comment:
Can I check if particular message (like TestMessage with key="x" and value="y") was published?
Please make sure that it supports generated classes (`org.apache.avro.specific.SpecificRecordBase`) from schema (`.avsc`)